controllers/upgrade.go

```diff
@@ -479,6 +497,16 @@ func (r *RollingUpgradeContext) SelectTargets(scalingGroup *autoscaling.Group) [
 	return targets
 }
 
```

Comment:
no need to add this func.
we already had https://github.com/keikoproj/upgrade-manager/blob/84dd4a7d7bd7b0f86bf3969d0821bf920ef27510/controllers/common/utils.go#L25

Reply:
Thanks for pointing out, I will use that

Reply:
fixed in [4511ff5](https://github.com/keikoproj/upgrade-manager/pull/394/commits/4511ff524f3093469ef17f72515d6d26f946b46c)